jupyter-notebooks/Data-API/search_and_preview_quickstart.ipynb

```diff
@@ -0,0 +1,496 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6db765fd-52c6-42d1-853c-4958a42f7099",
+   "metadata": {},
+   "source": [
+    "## Search & preview some imagery of farmland near Stockton, CA.\n",
```

Comment:
I imagine a new user would be looking at both this notebook and seach_and_download_quickstart.ipynb, trying to decide which to go for.  Having a consistent formatting in the titles and first paragraph would help make more clear what is similar and different between them.  Perhaps something like:

```suggestion
    "# Using Data API & Tile Service API to Efficiently Preview Imagery\n",
```